rust/processor/migrations/2024-05-21-221101_add_royalty_v1/up.sql

```diff
@@ -0,0 +1,24 @@
+-- Your SQL goes here
+CREATE TABLE token_royalty (
```

Comment:
Given we're moving away from tracking historical I don't think that we need this table.